bin/jurassic-tube-setup.sh

```diff
@@ -40,8 +43,26 @@ echo
 read -p "Please enter your Automattic/WordPress.com username: " username
 echo 
 
-${PWD}/docker/bin/jt/config.sh username ${username}
-${PWD}/docker/bin/jt/config.sh subdomain ${subdomain}
+if [ ! -f "${JT_DIR}/config.env" ]; then
+    touch "${JT_DIR}/config.env"
+else
+    > "${JT_DIR}/config.env"
+fi
+
+# Find the WordPress container section and get its port
+CLIENT_WORDPRESS_CONTAINER_SECTION=$(grep -A 20 "container_name: woocommerce_payments_wordpress" ${PWD}/docker-compose.yml)
+PORTS_LINE=$(echo "${CLIENT_WORDPRESS_CONTAINER_SECTION}" | grep "ports:" -A 1 | grep "\-")
+PORT=$(echo "${PORTS_LINE}" | grep -o '"[0-9]\+:' | cut -d'"' -f2 | cut -d: -f1)
+
+# Use default if extraction failed
+if [ -z "$PORT" ]; then
+    PORT=8082  # Default fallback
+    echo "Could not extract WordPress container port, using default: ${PORT}"
+fi
```

Comment:
Getting the localhost port and falling back to 8082 if an attempt to retrieve it was unsuccessful